domain-model-assistant/ProjectSettings/ProjectVersion.txt

```diff
@@ -1,2 +1,2 @@
-m_EditorVersion: 2021.2.9f1
-m_EditorVersionWithRevision: 2021.2.9f1 (921b45a28ab6)
+m_EditorVersion: 2022.1.18f1
+m_EditorVersionWithRevision: 2022.1.18f1 (8a091f9adba4)
```

Comment:
At the time I last pushed, this was the latest version, but that is no longer true. If you don't have this version, update your local Unity installation to the latest (currently 2022.1.20), convert the project, and see if it still works